src/StoreKeeper/WooCommerce/B2C/Exports/OrderExport.php

```diff
@@ -302,7 +302,8 @@ protected function processItem($WpObject)
         // Check if we should update the status
         if ($this->shouldUpdateStatus($storekeeper_status, $woocommerce_status)) {
             if (self::STATUS_REFUNDED === $woocommerce_status) {
-                // This will change the status of order to refunded
+                // This will change the status of order to refunded without sending payment ids
+                // Will only be a fallback if refunding processPaymentsAndRefunds does not change the status to refunded
```

Comment:
I don't really understand this sentence. why `processPaymentsAndRefunds` should not change it?

Reply:
Handling the worst case scenario, just like before that I called the `refundAllOrderItems`, and the status doesn't change to `refunded` but the refund is already attached to the order. We can remove this if we are really sure this will never happen again? :sweat_smile: